modules/InboundEmail/InboundEmail.php

```diff
@@ -3607,7 +3607,7 @@ public function isMailBoxTypeCreateCase()
         return ($this->mailbox_type == 'createcase' && !empty($this->groupfolder_id));
     } // fn
 
-    public function handleCreateCase($email, $userId)
+    public function handleCreateCase(Email $email, $userId)
```

Comment:
@jack7anderson7 in the scope of this change we should also cast the known calls to this method. To avoid getting the same issue in other places.

From a quick search I was able to find:
- tests/unit/phpunit/modules/InboundEmail/InboundEmailTest.php line 1727
- modules/Schedulers/_AddJobsHere.php line 666

Reply:
Hi @kgaillot,

Following @clemente-raposo response, the call in `_AddJobsHere.php` on line 666. We believe a check should be added in, see below;

```php
 if (empty($aopInboundEmailX->email)){
   throw new Exception('Invalid Email Type');
 }
```

We can add this change in or, if you would like to add the check or something similar then feel free.

Thanks,
Jack

Reply:
@clemente-raposo @jack7anderson7 Thanks, rebased on current hotfix branch and updated commit as suggested